rclrs/src/node/subscription.rs

```diff
@@ -125,6 +126,21 @@ where
         })
     }
 
+    /// Returns the topic name of the subscription.
+    ///
+    /// This returns the topic name after remapping, so it is not necessarily the
+    /// topic name which was used when creating the subscription
+    pub fn get_topic(&self) -> Result<Cow<str>, RclrsError> {
+        // SAFETY: No preconditions for the function used
+        // The unsafe variables get converted to safe types before being returned
+        unsafe {
+            let raw_topic_pointer = rcl_subscription_get_topic_name(&*self.handle.lock());
+            return Ok(CStr::from_ptr(raw_topic_pointer)
+                .to_string_lossy()
+                .to_owned());
+        }
+    }
```

Comment:
```suggestion
    pub fn topic_name(&self) -> String {
        // SAFETY: No preconditions for the function used
        // The unsafe variables get converted to safe types before being returned
        unsafe {
            let raw_topic_pointer = rcl_subscription_get_topic_name(&*self.handle.lock());
            CStr::from_ptr(raw_topic_pointer)
                .to_string_lossy()
                .into_owned()
        }
    }
```

Reply:
Same for the publisher. Rust doesn't need `return` at the end of the function, and `into_owned()` returns a String.

Reply:
I've removed the return statement... But The compiler is asking me to add `to_string()` after `into_owned()`... So I've added that in as well... Please let me know if I should have done it differently

Reply:
Only one thing is missing: please remove the `Result` and `Ok(…)`, since they're not needed.